src/features/wallet/scan/ScanView.tsx

```diff
@@ -110,68 +113,70 @@ const ScanView = () => {
       <Box flex={0.7} justifyContent="center" alignItems="center">
         <Crosshair width={wp(65)} height={wp(65)} color="white" />
       </Box>
-      <BottomSheet snapPoints={[300, 500]} initialSnapIndex={0}>
-        <Box>
-          <Text
-            variant="subtitleBold"
-            color="black"
-            fontSize={18}
-            marginBottom="s"
-          >
-            Find a QR Code to scan
-          </Text>
-          <Box marginBottom="s">
-            <Text
-              variant="subtitleBold"
-              color="black"
-              fontSize={16}
-              marginBottom="xxs"
-            >
-              Send HNT
-            </Text>
-            <Text marginBottom="xs">
-              HNT can be burned into Data Credits to pay for device network
-              connectivity
-            </Text>
-            <Text variant="body2Bold" color="blueMain">
-              Learn More
-            </Text>
-          </Box>
-          <Box marginBottom="s">
-            <Text
-              variant="subtitleBold"
-              color="black"
-              fontSize={16}
-              marginBottom="xxs"
-            >
-              Burn HNT to DC
-            </Text>
-            <Text marginBottom="xs">
-              HNT can be burned into Data Credits to pay for device network
-              connectivity
-            </Text>
-            <Text variant="body2Bold" color="blueMain">
-              Learn More
-            </Text>
-          </Box>
-          <Box marginBottom="s">
+      <BottomSheet snapPoints={[260, 400]} handleComponent={BSHandle}>
+        <BottomSheetScrollView style={{ padding: spacing.m }}>
+          <Box>
             <Text
               variant="subtitleBold"
               color="black"
-              fontSize={16}
-              marginBottom="xxs"
+              fontSize={18}
+              marginBottom="s"
             >
-              View my QR code
-            </Text>
-            <Text marginBottom="xs">
-              HNT can be burned into Data Credits to pay for device network
-              connectivity
-            </Text>
-            <Text variant="body2Bold" color="blueMain">
-              Learn More
+              Find a QR Code to scan
```

Comment:
There are some missing translations here.

Reply:
yeah these are still just yolo placeholders. I just changed the bottom sheet wrapping. Will move to translations once this is finalized